spec.bs

```diff
@@ -439,17 +448,18 @@ To <dfn>append private state token issue request headers</dfn> given a [=/reques
  1. If associating |issuer| with |topLevel| [=determine whether associating an issuer would exceed the top-level limit|would exceed the top level’s number-of-issuers limit=], return.
  1. [=Associate the issuer=] |issuer| with |topLevel|.
  1. If the [=number of tokens=] for |issuer| is at least 500, return.
- 1. Let |commitments| be the result of [=look up the key commitments|looking up the key commitments=] for |issuer|.
- 1. If |commitments| is null,  return.
+ 1. Let |issuerKeys| be the result of [=look up the key commitments|looking up the key commitments=] for |issuer|.
+ 1. If |issuerKeys| is null,  return.
  1. Let |signingKey| be the result of [=look up the latest keys|looking up of the latest keys=] for |issuer|.
  1. Run [=discard tokens=] with |issuer| and |signingKey|.
  1. Let |numTokens| be |issuer|'s [=max batch size=] or an [=implementation-defined=] limit on the number of tokens (which is recommended to be 100), whichever is smaller.
 
- 1. Let |tokens| be the result of [=generate masked tokens|generating masked tokens=] with |numTokens|.
+ 1. Let (|issueHeader|, |pretokens|) be the result of [=generate masked tokens|generating masked tokens=] with |issuerKeys| and |numTokens|.
  1. Set |request|'s [=cache mode=] to `"no-store"`.
- 1. Let |base64-encoded-tokens| be the base64-encoded version of |tokens|.
+ 1. Set |request|'s <dfn>pstPretokens</dfn> to |pretokens|.
+ 1. Let |base64EncodedTokens| be the base64-encoded version of |issueHeader|.
```

Comment:
Can you add https://www.rfc-editor.org/rfc/rfc4648 to the biblio at the top and reference it in this line?